screens/Profile.js

```diff
@@ -0,0 +1,70 @@
+import EStyleSheet from "react-native-extended-stylesheet";
+import Container from "../common/Container";
+import { View, Text, Image, TouchableOpacity } from "react-native";
+
+
+
+
+function Profile({ navigation, route }) {
+
+    return (
+        <Container>
+            <View style={styles.back}>
```

Comment:
Again, use absolute positioning here. Will make life easier when more stuff is added to this screen.